torch/distributed/_composable/fsdp/__init__.py

```diff
@@ -1,8 +1,3 @@
-from torch.distributed.fsdp import (
```

Comment:
Internal only: the before way meant that `from torch.distributed._composable.fsdp import fully_shard` was importing `fully_shard.py` not the function `fully_shard`. For some reason, the resolution order is different from open source.

To fix this, we match the old import as closely as possible. Namely, we import `fully_shard.py` contents from `.fully_shard`. This should force that import to take precedence.